src/Model/ActivityList/DefaultMariaDbActivityList/MySqlDao.php

```diff
@@ -150,4 +152,13 @@ protected function addOrder(QueryBuilder $select)
 
         return $this;
     }
+
+    public function configure(): void
+    {
+    }
+
+    public function setModel(AbstractModel $model): static
+    {
+        return $this;
```

Comment:
When we use the setModel from DaoInterface, then we should to a right implementation and also set the model?

```suggestion
        $this->model = $model;
        
        return $this;
```

Reply:
Yes 👍